partner_contact_department/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# © 2014-2015 Tecnativa S.L. - Jairo Llopis
+# © 2016 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    department_id = fields.Many2one("res.partner.department", "Department")
+
+
+class ResPartnerDepartment(models.Model):
```

Comment:
Please put this model in separate file.

Reply:
please note there is a mistake with your module naming: /ocabot migration contact_partner_department
it should be /ocabot migration partner_contact_department